webapp/src/ts/services/enketo.service.ts

```diff
@@ -355,10 +355,17 @@ export class EnketoService {
         window.history.back();
         this.setupNavButtons($wrapper, form.pages._getCurrentIndex() - 1);
         this.forceRecalculate(form);
+        this.pauseMultimedia($wrapper);
         return false;
       });
   }
 
+  private pauseMultimedia($wrapper) {
```

Comment:
Could we leave a comment here with a link back to the Enketo issue: https://github.com/enketo/enketo-core/issues/816

(Would make it a lot easier for future-us to remember that this can be removed eventually.)

Reply:
That's good idea!